tests/index.js

```diff
@@ -6,9 +6,13 @@ import {
 // ensure we don't mess up classic imports
 const CopyWebpackPlugin = require('./../dist/index');
 
-import fs from 'fs';
+import fs from 'fs-extra';
 import path from 'path';
 import _ from 'lodash';
+import chai from 'chai';
```

Comment:
Please no `chai` => [`jest.mock`]( http://facebook.github.io/jest/docs/en/mock-function-api.html#content) 🙃

Reply:
@michael-ciniawsky i think we do this when merge with default

Reply:
hmm... 😞  fair point but it's still needs to be refactored again with `webpack-defaults` then 🤢. Can't be helped